<?xml version="1.0"?>
<launch>

      <!-- bridge nodes-->
      <node pkg="hero_bridge" type="speech.py" name="text_to_speech" if="$(optenv ROBOT_REAL false)" output="log" />
      <node pkg="hero_bridge" type="gripper_controller.py" name="gripper_bridge" output="log" />
      <node pkg="hero_bridge" type="tf_ghost.py" name="tf_ghost" output="log" />
      <node pkg="hero_bridge" type="head_controller.py" name="head_bridge" output="log" />
      <node pkg="hero_bridge" type="handover_detection.py" name="handoverdetector" output="log" />

      <include file="$(find hero_bridge)/launch/manipulation_bridge.launch" />

</launch>
